src/SearchField/_variables.scss

```diff
@@ -5,7 +5,8 @@ $search-btn-variants: (
 $search-border-radius:                0 !default;
 $search-line-height:                  1.5rem !default;
 $search-border-color:                 $gray-500 !default;
-$search-form-background-color:        $white !default;
+$search-label-background-color:        $white !default;
+$search-button-background-color:        $white !default;
```

Comment:
Please align these two variables, similar to others in this file
![image](https://github.com/openedx/paragon/assets/93188219/7a9a76aa-1ca5-4e7f-b888-262c577da94f)

Reply:
It will also be necessary to add a new token design to the alpha branch. For a new variable: `$search-button-background-color`.

Reply:
I`ve aligned variables, tokens will be added after the merge.